src/Pos/Lrc/DB/Issuers.hs

```diff
@@ -19,7 +19,7 @@ import           Pos.DB.Class      (MonadDB)
 import           Pos.DB.Error      (DBError (DBMalformed))
 import           Pos.Lrc.DB.Common (getBi, putBi)
 import           Pos.Types         (Coin, EpochIndex, EpochIndex (..), StakeholderId)
-import           Pos.Util          (maybeThrow)
+import           Pos.Util     (maybeThrow)
```

Comment:
spokuha, it wouldn't help in this case, because i have replaced `Pos.Util.Util` in all files at once